index.go

```diff
@@ -208,7 +208,7 @@ func readIndex(r io.Reader) ([]IndexKey, error) {
 			if len(fingerprintSlice) != 20 {
 				return keys, errors.New("hkp: invalid fingerprint size")
 			}
-			var fingerprint [20]byte
+			fingerprint := make([]byte, 20)
 			copy(fingerprint[:], fingerprintSlice)
```

Comment:
Now that the fingerprint is a slice instead of an array, I don't think we need this `copy` call anymore - we can use `fingerprintSlice` directly (and ideally rename it to `fingerprint`).